lints/base.go

```diff
@@ -216,10 +221,24 @@ func (report *LintReport) Execute(cert *x509.Certificate) error {
 			return err
 		}
 		l.updateReport(report, res)
+		updateErrorStatePresent(report, res)
 	}
 	return nil
 }
 
+func updateErrorStatePresent(report *LintReport, result ResultStruct) {
```

Comment:
I'm not sure why this is here either. Why isn't all of this code in the protobuf thing that you generate given that's the location we need it?